angular-workspace/projects/ni/nimble-angular/test.ts

```diff
@@ -30,7 +32,6 @@ context.keys().map(context);
 
 // Workaround to make console.error calls fail tests:
 // https://github.com/angular/angular/issues/36430#issuecomment-874772398
-// Might be removable after upgrading to Angular 14:
-// https://github.com/angular/angular/issues/36430#issuecomment-1117801535
+// This can be useful to have even with errorOnUnknownElements and errorOnUnknownProperties set to true.
 // eslint-disable-next-line no-console, @typescript-eslint/no-explicit-any
 console.error = (data: any): void => fail(data);
```

Comment:
This seems generally useful, we should consider a PR update to the [new app handbook](https://dev.azure.com/ni/DevCentral/_git/Skyline?path=/docs/developer-handbook/How-to-guides/How-to-generate-an-application-for-development.md&_a=preview&version=GBmaster&anchor=complete-the-repository) and enabling it across the projects in SystemLinkShared.
Can you make a tech debt item with those tasks in the same style as the [errorOnUnknownElement](https://dev.azure.com/ni/DevCentral/_workitems/edit/2446157) to track it (and reference that tech debt item from here)

Reply:
Created work item: https://dev.azure.com/ni/DevCentral/_workitems/edit/2472903